### ISSUE TITLE
Does this support hand tracking from the Leap Motion as well? + Can we have a tutorial

### ISSUE THREAD

**Issue by NataIynn** [adeleine1412/leapmotion-osc#5]
Does this support hand tracking?

Can we also have a tutorial on this :)? I believe I set up everything correctly, but I don't see my hands moving nor fingys.

**Comment by philippgitpush** [comment]
This script only allows finger tracking, for hand tracking you would have to strap tracker or controller to hand/wrist for now. Positioning two hands in 3D space with OSC will be difficult, or at least out of my reach.

I really recommend you look into the Leapmotion SteamVR driver instead, which would theoretically allow you to have both hand and finger tracking in many other games besides VRChat. (It will also look much smoother because OSC currently has a limited sync rate, your fingers would move well four you but stutter for other people when you use OSC)

tl;dr this is more of a proof of concept.

The SteamVR-Wide driver for Leapmotion, which also offers options for finger controls, can be found here:
https://github.com/SDraw/driver_leap
